src/Polymath.ts

```diff
@@ -431,6 +434,35 @@ export class Polymath {
     return await procedure.prepare();
   };
 
+  public cancelSTO = async (args: {
+    moduleUid:
+      | {
+          securityTokenId: string;
+          stoType: StoModuleTypes;
+          address: string;
+        }
+      | string;
+    value: BigNumber;
+    custodianAddress: string;
+  }) => {
+    let securityTokenId: string; let stoType: StoModuleTypes; let address: string; let symbol: string;
+    if (typeof args.moduleUid === 'string') {
```

Comment:
As per my suggestion above, this check shouldn't be necessary since this will always be a string